src/tests/test_cases/optional.py

```diff
@@ -0,0 +1,26 @@
+"""This is a module."""
+
+from .expected import Expectation
+
+expectation = Expectation()
+expect = expectation.expect
+
+
+@expect('D212: Multi-line docstring summary does not start on the first line.')
+def multi_line_starts_second_line():
+    """
```

Comment:
could you add some tests that use u"""...""" , r"""...""" ,etc.  I suspect that `if first not in ['"""', "'''"]:` wont catch them all.